src/utils/prompt.ts

```diff
@@ -58,14 +58,19 @@ export const prompt = {
     return inquirerConfirm.default(config, { output: errorOutput === 'stderr' ? process.stderr : process.stdout });
   },
 
-  /* c8 ignore next 10 */
+  /* c8 ignore next 15 */
   async forSelection<T>(config: SelectionConfig<T>): Promise<T> {
     if (!inquirerSelect) {
       inquirerSelect = await import('@inquirer/select');
     }
 
     const cli = Cli.getInstance();
     const errorOutput: string = cli.getSettingWithDefaultValue(settingsNames.errorOutput, 'stderr');
+    const promptPageSizeCap: number = cli.getSettingWithDefaultValue(settingsNames.promptListPageSize, 7);
```

Comment:
TBH I liked the previous default implementation. Now when user will like to have all options visible then he or she would need to change the setting to something really high like 100 🤔.

Reply:
Well, autocomplete never has a 100 items of course. Anyway, if you make it a true/false setting you still need another setting with a number to configure the limit. I think just using a capsize is better. There's no way (I believe) people want to set this as high as a 100. That would be a very bad user experience I think.

Reply:
What do you mean with "the previous default configuration"...

Reply:
What I've added is not just for autocomplete by the way, it now concerns a configuration for all list prompts. If more than x items are in the list, it will paginate the list. I think a configurable limit is the best way to go as we're a terminal tool and we're therefore bound to limits anyway. no-one likely would want to scroll through a list of 50 items in the terminal. That would probably be very bad UX. 

I've picked the default limit for inquirer because I thought he probably did that for a reason as well. But I'm perfectly fine with making it a little higher, say 10 or 15.

I think this is better than a true/false configuration for pagination. As that would make you want to configure the pagination size as well.

Reply:
I wasn't thinking of a true/false setting 🙂. Sorry if I was unclear.
My proposal was to:
1. if the setting `promptListPageSize` is not set (not customized by the user just left as is after installing CLI) then the default would be to always print all choices. so 'as before' print the autocomplete.length.
2. if needed the user may change this property to be a static number (like a fixed 7)